content/authentication/connecting-to-github-with-ssh/adding-a-new-ssh-key-to-your-github-account.md

```diff
@@ -75,7 +75,7 @@ To include a title for the new key, use the `-t` or `--title` flag.
 gh ssh-key add KEY-FILE --title "personal laptop"
 ```
 
-If you generated your SSH key by following the instructions in "[AUTOTITLE](/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)", you can add the key to your account with this command.
+If you generated your SSH key by following the instructions in "[AUTOTITLE](/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent)", you can add the key to your account with this command. If you're prompted to request additional scopes, follow the instructions in the command line.
```

Comment:
Sorry, may have spoken too soon 🤦 I think it might make sense to add the sentence to the end of line 66 instead

Reply:
No problem, I misunderstood the problem myself too.
I have updated the PR and also the screenshots. 
Feel free to take a look whenever you get a chance.

Thanks!